demo/src/app/components/+datepicker/demos/config-method/config-method.html

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-sm-4 form-group">
+    <input type="text" class="form-control" #dp="bsDatepicker" bsDatepicker [bsConfig]="bsConfig">
```

Comment:
[bsConfig] binding isn't needed since you're setting datepicker options via `setConfig()` method